### PR TITLE
Update UI, and text for Contracting documents section

### LINKED ISSUE #3458: Update Contracting documents section
Fixes in the Contracting documents section:

- [ ] Replace 'Contracting team' by 'org_short_name' in text.
- [ ] Section update, margin, and tick marks.
- [ ] Update text, add 'Pending' and other things.
- [ ] Final tick to Contract only after submission of contracting documents to staff.
- [ ] Update next step for applicant/Vendor.
image
image

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Fixes #3458  Refs: #3448

For staff:

image

For Vendor:
image

After contract and document upload:
for staff:
image

for vendor:
image

### CODE REVIEW

**Comment by theskumar** [comment]
@sandeepsajan0 Can you please a screenshot of the UI changes?

**Comment by sandeepsajan0** [comment]
@theskumar I have updated the description with changed UI screenshots. It is as per the design suggested in figma.

@abhiXII you may also have a look at the screenshots and confirm if everything seems fine. 
I left a horizontal line implementation between signed and countersigned statements because I think last time we removed it so that it can look like one body or section. Ex: Approval Form and SOW Form.

File: hypha/apply/projects/templates/application_projects/includes/contracting_documents.html [hunk 1]
```diff
@@ -50,22 +50,11 @@ <h5> {% trans "Are you sure you want to submit contracting documents?" %}</h5>
             </div>
             {% if contract_uploaded %}
             <div class="docs-block__row-inner">
-                <a class="docs-block__icon-link" href="{% url 'apply:projects:contract' pk=project.pk file_pk=contract.pk %}" target="_blank">
+                <a class="font-bold flex items-center w-auto text-left bg-white text-light-blue mr-0 mb-1 p-2.5 border-solid border border-light-blue" href="{% url 'apply:projects:contract' pk=project.pk file_pk=contract.pk %}" target="_blank">
```
Comment (theskumar) [inline hypha/apply/projects/templates/application_projects/includes/contracting_documents.html]: As the button is quite a re-usable UI element consider using `button button--white button--half-width` or it's variation to keep the look consistent. 

Also, the button styling misses focus and hover state. 

Screenshot 2023-07-11 at 04 29 27@2x

**Review by theskumar — CHANGES_REQUESTED** [review]

File: hypha/apply/projects/views/project.py [hunk 2]
```diff
@@ -319,6 +319,7 @@ def get_context_data(self, **kwargs):
         context['contract_to_sign'] = contract_to_sign
         context['contracts'] = contracts.approved()
         context['contract'] = latest_contract
+        context['org_short_name'] = settings.ORG_SHORT_NAME
```
Comment (frjo) [inline hypha/apply/projects/views/project.py]: Do we need to set this here when we have it in:

https://github.com/HyphaApp/hypha/blob/bf4cd5416d32cb43a358bd202067a63c02378035/hypha/core/context_processors.py#L6-L21

File: hypha/apply/projects/templates/application_projects/includes/contracting_documents.html (hunk 1 above)
Comment (sandeepsajan0) [reply hypha/apply/projects/templates/application_projects/includes/contracting_documents.html]: I agree, we have project-actions--white as well but this is a custom button, we have a few other custom buttons as well like Upload, and Reupload contracts. These buttons look a bit different(more padding) from others just to get the user's attention.
 
Actually, these have to be links like other View options but these actions are slightly more important than links and less important than buttons.

I have added focus and hover state with bg and text colour change.

**Comment by sandeepsajan0** [comment]
The hover state for the icon is creating issues for other 'View' links. I'll fix it.
https://github.com/HyphaApp/hypha/blob/4a1255d4f7a810c3f2a4ebd083849e9c08a07fa8/hypha/static_src/src/sass/apply/components/_icon.scss#L126-L128

**Comment by abhiXII** [comment]
Instead of contracting team, we're going to say OTF but I think this is editable on Wagtail so we can do that. 
without the line looks fine too. Thanks

Very minors things I noticed for consistency were: the (i) icon with `corrections/ammendments?' should also be grey and the (i) icon with view template should be on the left of view template text. so that all (i) icons look and feel same on the page.

Also I believe we're not using time anywhere? I'm okay with it being left there but you can see as per the system consistency.

**Comment by sandeepsajan0** [comment]
>Instead of contracting team, we're going to say OTF

It is OTF for vendors/applicants, 'contracting team' is only for staff/team.

I have made the info (i) icon consistent.  For now, let's keep time for the contracts, we can change it anytime if needed. it is just a small change.

@frjo I think we can put its latest version to the test site.

**Comment by frjo** [comment]
Deploying last to test now.

**Comment by fourthletter** [comment]
@abhiXII - thank you for taking a look!

**Pull request merged**